timed/employment/admin.py

```diff
@@ -146,77 +144,87 @@ class AbsenceCreditInline(admin.TabularInline):
 class UserAdmin(UserAdmin):
     """Timed specific user admin."""
 
-    inlines = [
+    inlines = (
         SupervisorInline,
         SuperviseeInline,
         EmploymentInline,
         OvertimeCreditInline,
         AbsenceCreditInline,
-    ]
-    list_display = ("username", "first_name", "last_name", "is_staff", "is_active")
-    search_fields = ["username"]
+    )
+    list_display = (
+        "username",
+        "first_name",
+        "last_name",
+        "is_staff",
+        "is_active",
+    )
+    search_fields = ("username",)
 
-    actions = [
+    actions = (
         "disable_users",
         "enable_users",
         "disable_staff_status",
         "enable_staff_status",
-    ]
+    )
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.fieldsets += (
             (_("Extra fields"), {"fields": ["tour_done", "is_accountant"]}),
         )
 
-    def disable_users(self, request, queryset):
+    def disable_users(self, _, queryset):
         queryset.update(is_active=False)
 
     disable_users.short_description = _("Disable selected users")
 
-    def enable_users(self, request, queryset):
+    def enable_users(self, _, queryset):
         queryset.update(is_active=True)
 
     enable_users.short_description = _("Enable selected users")
 
-    def disable_staff_status(self, request, queryset):
+    def disable_staff_status(self, _, queryset):
         queryset.update(is_staff=False)
 
     disable_staff_status.short_description = _("Disable staff status of selected users")
 
-    def enable_staff_status(self, request, queryset):
+    def enable_staff_status(self, _, queryset):
         queryset.update(is_staff=True)
 
     enable_staff_status.short_description = _("Enable staff status of selected users")
 
-    def has_delete_permission(self, request, obj=None):
+    def has_delete_permission(self, _, obj=None):
```

Comment:
Instead of just taking `_` as parameter, you can use `_request` to denote that it's unused, but still document that it's actually a request. Alternatively, type hints

Reply:
i didn't know this were possible, its a lot more readable